samza-core/src/test/java/org/apache/samza/operators/TestMessageStreamImpl.java

```diff
@@ -18,6 +18,15 @@
  */
 package org.apache.samza.operators;
 
+import java.util.ArrayList;
```

Comment:
Codestyle: Here and in other RB too, I think the current order b/w org.* and java.* is as it was earlier. Let's make sure it's consistence w/ the existing code.